vcr/issuer/openid4vci/issuer.go

```diff
@@ -233,7 +246,11 @@ func (i *issuer) HandleCredentialRequest(ctx context.Context, issuer did.DID, re
 // validateProof validates the proof of the credential request. Aside from checks as specified by the spec,
 // it verifies the proof signature, and whether the signer is the intended wallet.
 // See https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#name-proof-types
-func (i *issuer) validateProof(request oidc4vci.CredentialRequest, issuer did.DID, wallet did.DID) error {
+func (i *issuer) validateProof(ctx context.Context, flow *Flow, request oidc4vci.CredentialRequest) error {
+	credential := flow.Credentials[0] // there's always just one (at least for now)
```

Comment:
a Credential Request is always for 1 credential at a time. The Batch Request Endpoint is for multiple credentials in a single request

Reply:
ok, so?

Reply:
nvm. copied line